### DISCUSSION TITLE
Azure Function API Oauth2

### DISCUSSION THREAD

**Discussion by mschweig** [grafana/grafana-infinity-datasource#911] [Q&A] [answered]
Hi, 

I tried to configure the Oauth2 Authentication to request Azure Function APIs that are secured via Azure AD / Entra ID. I set the correct ClientID, but I am getting 401 Unauthorized when testing with the Function API URL. 

So my question is how to properly configure the App Registration and the Plugin to get access to the Functions? 

Thanks 
Best regards, 
Manuel

**Comment by mschweig** [comment, marked as answer]
After Setting the proper scope with ./default postfix the Auth with Entra ID is perfectly working

**Reply by Wesztman** [reply]
@mschweig , what was the proper scope? :) I'm also stuck on 401 :p
